doc/modules/impute.rst

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
You need to remove this content from the preprocessing docs. You're welcome to leave a "see also" reference to these docs in preprocessing if you think that will help users.